app/assets/stylesheets/school-onboarding-profile.scss

```diff
@@ -0,0 +1,11 @@
+#school-onboarding-profile {
```

Comment:
Not really sure why this is here - can it be removed? 

The `#school-onboarding-profile` is on the existing summary page but the `.button-group-container` is on the new preview page

Reply:
Nice spot. This was left over from when there was a button added to preview the profile, rather than it being the next step after check answers, removed it 👍